libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -50,6 +50,21 @@ namespace SohImGui {
     Console* console = new Console;
     bool p_open = false;
     bool needs_save = false;
+    float kokiri_col[3] = { 0.118f, 0.41f, 0.106f };
```

Comment:
Is there a type predefined for colour? Or is the same type used in decomp for colour? Color_RGB8.
Probably fine like this since you save it as a float. But you probably could just do colorrgb8.